src/components/core/DpEditor/DpEditor.stories.mdx

```diff
@@ -8,12 +8,46 @@ import DpEditor from './DpEditor'
 
 # Editor
 
-A rich text editor component based on tiptap.
+The Editor component renders a rich text editor consisting of a menu bar supporting common editor interactions and a text field.
 
-- defines a menubar with a number of buttons and an editor
-- use this component without the inline-editing-wrapper TiptapEditText.vue if you want to add a text editor to a form element (as in new statement view)
-- use this component with the inline-editing-wrapper TiptapEditText.vue if you want to save the text directly via inline-editing (as in assessment table)
-- the editor-content component needs a prop with editor instance to work correctly
 
-To properly set the content we have to update this.currentValue and use editor.setContent() - both actions are needed!
-To use boilerplates we need to mount tiptap editor, where boilerplate modal is imported dynamically. The boilerplates are stored in boilerplates vuex store, which is also registered dynamically if boilerplate prop is specified.
+To use image upload we need to add imageButton optional Prop, and when using `imageButton` Prop, you have to set also
+`routes.getFileByHash` Prop.
+
+To use another component within DpEditor, we can add Slot.
+
+## Default usage
+
+```html
+<dp-editor
+    :value="value"
+    ref="r_links">
+</dp-editor>
+```
+
+## Custom usage
+
+```html
+<dp-editor
+    :toolbar-items="{
+        fullscreenButton: true,
+        insertAndDelete: true,
+        table: true,
+        headings: [2,3,4],
+        imageButton: true,
+        linkButton: true,
+        strikethrough: true
+    }"
+    :routes="{
+        getFileByHash: (hash) => RoutingFunction(hash)
+    }"
+    :value="value">
+    <template v-slot:modal="modalProps">
+        <dp-modal
+          id="openModal"
+          ref="openModal">
+          Add an component
+        </dp-modal>
+    </template>
```

Comment:
Well, if we really want to document this case here, we'd have to show it all, including the usage of `handleInsertText` exposed by DpEditor, and the button which triggers the modal.

```suggestion
    <template v-slot:modal="modalProps">
        <dp-modal
            id="editorModal"
            ref="editorModal"
            @insert="text => modalProps.handleInsertText(textFromParentComponent)">
            Add an component
        </dp-modal>
    </template>
    <template v-slot:button>
        <dp-button
              :text="Add Text"
              variant="subtle"
              @click.stop="openEditorModal" />
    </template>
```